src/components/ValidatedInput/ValidFieldTextArea.scss

```diff
@@ -15,6 +15,9 @@
     font-size: $font-size-h6;
     padding-bottom: 4px;
   }
+  .field-error-message {
+    font-size: $font-size-h7;
```

Comment:
if we only change it here, there will be inconsistency among error message in different input components.